src/plugins/subsystems/drm/drm.sh

```diff
@@ -208,22 +216,41 @@ function gui_control()
   local isolate_target
   local remote
   local port
+  local set_default=false
+  local default_command
 
   flag=${flag:-'SILENT'}
 
   if [[ "$operation" == 'ON' ]]; then
     isolate_target='graphical.target'
     vt_console=1
     gui_control_cmd="${configurations[gui_on]}"
-  else
+  elif [[ "$operation" == 'OFF' ]]; then
     isolate_target='multi-user.target'
     vt_console=0
     gui_control_cmd="${configurations[gui_off]}"
+  elif [[ "$operation" == 'ON_AFTER_REBOOT' ]]; then
+    isolate_target='graphical.target'
+    vt_console=1
+    gui_control_cmd="${configurations[gui_on_after_reboot]}"
+    set_default=true
+  elif [[ "$operation" == 'OFF_AFTER_REBOOT' ]]; then
+    isolate_target='multi-user.target'
+    vt_console=0
+    gui_control_cmd="${configurations[gui_off_after_reboot]}"
+    set_default=true
   fi
 
   # If the user does not override the turn on/off command we use the default
   # systemctl
-  gui_control_cmd=${gui_control_cmd:-"systemctl isolate ${isolate_target}"}
+
+  if $set_default; then
```

Comment:
I actually didn't know that bash understood the keywords `true` and `false` :rofl:

However, I think it is more reliable to explicitly check to see if the value of `set_default` is `true`, i.e.,

```bash
`if [[ "$set_default" == 'true' ]]`; then
```

Reply:
:+1:  done